software/opensign.yml

```
@@ -0,0 +1,12 @@
+name: "OpenSign"
+website_url: "https://www.opensignlabs.com"
+source_code_url: "https://github.com/opensignlabs/opensign"
+description: "Free, open source & self-hosted document signing software. Alternative to DocuSign."
```

Syntax for alternatives can be found here:https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

```suggestion
description: "Free, open source & self-hosted document signing software (alternative to DocuSign)."
```